### DISCUSSION TITLE
Get rid of the welcome message

### DISCUSSION THREAD

**Discussion by sim186** [JanDeDobbeleer/oh-my-posh#2252] [Troubleshoot]
Hi all,

is there a way to get rid of the welcome message?
![image](https://user-images.githubusercontent.com/18590835/167866807-f42c3fb9-ce7b-4fb0-b687-2b2706f7227e.png)

**Comment by JanDeDobbeleer** [comment]
Yes, by following the migration guide that's mentioned in the message 😃

**Reply by rinshadm** [reply]
I followed the migration guide and still couldn't get rid of the message.
Finally deleted the message from `(Get-Module oh-my-posh).Path` and it worked.

**Reply by JanDeDobbeleer** [reply]
@rinshadm this means you did not uninstall the module:

```powershell
Uninstall-Module oh-my-posh -AllVersions
```

**Reply by rinshadm** [reply]
@JanDeDobbeleer yes you're right. Now I uninstalled the module and replaced `Set-PoshPrompt` as per the migration guideline. Things are working fine. Thank you.

**Reply by dongup** [reply]
You could add "clear" command to the end of powershell profile file (.ps1 file).
```
Import-Module posh-git
Import-Module oh-my-posh
Set-PoshPrompt -Theme blue-owl
oh-my-posh init pwsh --config 'https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/blue-owl.omp.json'
clear
```

**Reply by JanDeDobbeleer** [reply]
@dongup you need to remove the `Import-Module oh-my-posh` statement. That's no longer needed/supported. Have a look [here](https://ohmyposh.dev/docs/migrating).

**Reply by dongup** [reply]
> guide

ahh, I did it, thank you very much.
